tests/test_profile.py

```diff
@@ -0,0 +1,170 @@
+import pytest
+
+import json
+import os
+import shutil
+
+from great_expectations.profile.base import DataSetProfiler
+from great_expectations.profile.basic_dataset_profiler import BasicDatasetProfiler
+from great_expectations.profile.columns_exist import ColumnsExistProfiler
+from great_expectations.dataset.pandas_dataset import PandasDataset
+from great_expectations.data_context import DataContext
+from great_expectations.data_context.util import safe_mmkdir
+
+# Tests to write:
+# test_cli_method_works  -> test_cli
+# test context-based profile methods
+# test class-based profile methods
+
+
+def test_DataSetProfiler_methods():
+    toy_dataset = PandasDataset({"x": [1, 2, 3]})
+
+    assert DataSetProfiler.validate_dataset(1) == False
+    assert DataSetProfiler.validate_dataset(toy_dataset)
+
+    with pytest.raises(NotImplementedError) as e_info:
+        DataSetProfiler.profile(toy_dataset)
+
+
+def test_ColumnsExistProfiler():
+    toy_dataset = PandasDataset({"x": [1, 2, 3]})
+    expectations_config, evr_config = ColumnsExistProfiler.profile(toy_dataset)
+
+    print(json.dumps(expectations_config, indent=2))
```

Comment:
This isn't testing anything

Reply:
smoke test---verifies that it doesn't crash.

And yes, it should also have some asserts, too.

Reply:
kk